man/man7/zfsprops.7

```diff
@@ -900,13 +900,16 @@ zeroes (the NUL byte).
 When a zero-filled block is detected, it is stored as
 a hole and not compressed using the indicated compression algorithm.
 .Pp
-Any block being compressed must be no larger than 7/8 of its original size
-after compression, otherwise the compression will not be considered worthwhile
-and the block saved uncompressed.
-Note that when the logical block is less than
-8 times the disk sector size this effectively reduces the necessary compression
-ratio; for example, 8kB blocks on disks with 4kB disk sectors must compress to 1/2
-or less of their original size.
+All blocks are allocated as a whole number of sectors
+(chunks of 2^ashift bytes, e.g. 512B or 4KB).
+Compression may result in a non-sector-aligned size, which will be rounded up
+to a whole number of sectors.
+If compression saves less than one whole sector,
+the block will be stored uncompressed.
+Therefore, blocks whose logical size is a small number of sectors will
+experience less compression ratio
```

Comment:
"a lesser compression ratio" or "less compression"?

Reply:
I'm not a native speaker, but `less compression ratio` looks good for me. `less compression` == `less compression ratio` for me.

Reply:
You can't have "less ratio", because ratio is in this case a property, which you either have or you don't (you could have "fewer ratios", but that'd mean something different). A ratio, by definition, can get bigger or smaller – so either "a smaller/lesser/&c. compression ratio" or just "less compression"; I'm more inclined toward just "less compression" because it's more obvious what that actually means.

Reply:
I didn't get everything, unfortunately, but I'm ok with "less compression", updated.